ccmlib/common.py

```diff
@@ -241,6 +242,18 @@ def make_cassandra_env(install_dir, node_path, update_conf=True):
     env['cassandra.config'] = "file://" + os.path.join(node_path, 'conf', 'cassandra.yaml')
     env['CASSANDRA_HOME'] = install_dir
     env['CASSANDRA_CONF'] = os.path.join(node_path, 'conf')
+
+    if hardcode_java_home_8:
+        output = subprocess.check_output(["update-alternatives", "--display", "java"], text=True).splitlines()
```

Comment:
i still don't like the way how we are using `update-alternatives`, this command is a user facing tool in the sense that its output format is supposed to be consumed by human. so the way to parse its output is fragile. i'd suggest use something like https://github.com/scylladb/scylla-jmx/blob/master/scripts/select-java .